centreon/www/front_src/src/Configuration/Commands/utils.ts

```diff
@@ -0,0 +1,21 @@
+import { Filters } from './models';
+
+export const defaultSelectedColumnIds = [
+  'name',
+  'command_line',
+  'host_uses',
+  'service_uses',
+  'type',
+  'actions',
+  'is_activated'
+];
+
+export const filtersInitialValues: Filters = {
+  name: '',
+  enabled: false,
+  disabled: false,
+  type: []
+};
+
+export const filtersAtomKey = 'filters_commands';
```

Comment:
Lets formalize keys using centreon_<page-or-feature>_<purpose>_<version>
```suggestion
export const filtersAtomKey = 'centreon_commands_filters_1';
```
The version ensure to not use an invalid version of the localstorage there is a breaking change